src/Examine.Lucene/Search/LuceneFacetOperationBase.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Examine.Search;
+
+namespace Examine.Lucene.Search
+{
+    public abstract class LuceneFacetOperationBase : IFaceting
```

Comment:
Do we need this base class? Seems like it is not directly used by anything

Reply:
It was just to match the other classes but you're right it doesn't serve any purpose. I'll remove it.